mungegithub/submit-queue/deployment/kubernetes/configmap.yaml

```diff
@@ -14,8 +14,11 @@ data:
   # test-options feature options.
   test-options.required-retest-contexts: "\"Jenkins GCE e2e\",\"Jenkins unit/integration\",\"Jenkins verification\",\"Jenkins GCE Node e2e\",\"Jenkins Kubemark GCE e2e\",\"Jenkins GCE etcd3 e2e\""
 
+  submit-queue.protected-branches: "master"
+  submit-queue.protected-branches-extra-contexts: "cla/linuxfoundation"
```

Comment:
Filed https://github.com/kubernetes/contrib/issues/2129. @eparis, for now, maybe we could exclude the CLA status from being marked as required perhaps?